release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -120,6 +120,16 @@ For {product-title} {product-version}, you can quickly install a cluster on Amaz
 
 For more information, see xref:../installing/installing_aws/installing-aws-localzone.adoc#installation-cluster-quickly-extend-workers_installing-aws-localzone[Intall a cluster quickly in AWS Local Zones].
 
+[id="ocp-4-14-install-update-cco-manual-enhancements"]
+==== Simplified installation and update experience for clusters with manually maintained cloud credentials
+
+This release includes changes that improve the experience of installing and updating clusters that use the Cloud Credential Operator (CCO) in manual mode for cloud provider authentication. The following parameters for the `oc adm release extract` command simplify the manual configuration of cloud credentials:
+
+`--included`:: Use this parameter to extract only the manifests that your specific cluster configuration needs. If you use cluster capabilities to disable one or more optional components, you are no longer required to delete the `CredentialsRequest` custom resources for any disabled components before installing or updating a cluster. This parameter also makes the CCO utility (`ccoctl`) `--enable-tech-preview` parameter unnecessary.
+
+`--install-config`:: Use this parameter to specify the location of the `install-config.yaml` file when installing a cluster. By referencing the `install-config.yaml` file, the extract command can determine aspects of the cluster configuration. With this change, you are no longer required to specify the cloud platform you are installing on with the `--cloud` parameter. As a result, the `--cloud` parameter is deprecated starting in {product-title} 4.14.
```

Comment:
"...aspects of the configuration of a cluster that you are about to create."?  Andmaybe something about not needing to set `--install-config` once a cluster exists, because `oc` can connect to the cluster to query its configuration?  Or leave that level of detail to the docs you link below?

Reply:
RNs is a good place for that - otherwise we end up talking about install in the update content or vice versa :)